rpm/tarantool.spec

```diff
@@ -196,8 +202,10 @@ make %{?_smp_mflags}
 # %%doc and %%license macroses are used instead
 rm -rf %{buildroot}%{_datarootdir}/doc/tarantool/
 
+%ifnarch aarch64
 %check
 make test-force
+%endif
```

Comment:
It is okay, but requires a comment that links a reason of the tests disabling (preferably an issue number).

Reply:
I would perform some very simple smoke test when we don'tt run a full testing, like we do in the [homebrew's formula](https://github.com/Homebrew/homebrew-core/blob/65b4518a9a36908e805ac85a7912f81df8523c9e/Formula/tarantool.rb#L71-L85). Feel free to ignore.

Reply:
Still actual.